src/Umbraco.Core/Configuration/Grid/GridEditorsConfig.cs

```diff
@@ -95,4 +141,10 @@ List<IGridEditorConfig> GetResult()
             return result!;
         }
     }
+
+    private static IFileInfo? GetConfigFile(IFileProvider fileProvider, string path)
+    {
+        IEnumerable<IFileInfo> contents = fileProvider.GetDirectoryContents(path);
+        return contents.FirstOrDefault(file => file.Name.InvariantEquals("grid.editors.config.js") && !string.IsNullOrEmpty(file.PhysicalPath));
```

Comment:
```suggestion
        IFileInfo fileInfo = fileProvider.GetFileInfo($"{path}/grid.editors.config.js");
        return fileInfo.Exists && fileInfo.PhysicalPath.IsNullOrWhiteSpace() == false
            ? fileInfo
            : null;
```

Not sure we need to iterate the directory contents when we know the file name - unless of course we want this to be case insensitive?